src/Metadata.elm

```diff
@@ -79,6 +79,12 @@ metadataFromPage page language content =
                 Nothing ->
                     defaultMetadata language
 
+        SubmitStory ->
+            { title = t SubmitStoryTitle
```

Comment:
These should come from the markdown now?

Reply:
I think I've done it using your existing page meta data code and a couple of tweaks.